graphql2/graphqlapp/query.go

```diff
@@ -173,6 +173,11 @@ func (a *Query) DebugMessages(ctx context.Context, input *graphql2.DebugMessages
 			return nil, fmt.Errorf("format dest: %w", err)
 		}
 
+		// notifications that end up bundled are omitted
+		if m.MessageType == notification.MessageTypeAlert && m.LastStatus == notification.StateBundled {
```

Comment:
This should be handled in the DB query so the limit applies **after** they are filtered out